plugins/woocommerce-admin/client/customize-store/index.tsx

```diff
@@ -163,10 +165,14 @@ export const customizeStoreStateMachineDefinition = createMachine( {
 			states: {
 				preIntro: {
 					invoke: {
-						src: 'fetchThemeCards',
+						src: 'fetchIntroData',
```

Comment:
To avoid errors causing white screen, could we add an `onError` handler for this? 🙂

Reply:
NVM :) I think I can use your https://github.com/woocommerce/woocommerce/pull/40430/files